LibreNMS/Util/ModuleTestHelper.php

```diff
@@ -191,7 +191,7 @@ private function collectOids($device_id)
         $collection_output = preg_replace('/\033\[[\d;]+m/', '', $collection_output);
 
         // extract snmp queries
-        $snmp_query_regex = '/SNMP\[.*snmp(?:bulk)?([a-z]+)\' .+(udp|tcp|tcp6|udp6):(?:\[[0-9a-f:]+\]|[^:]+):[0-9]+\' \'(.+)\']/';
+        $snmp_query_regex = '/^SNMP\[\'.*snmp(?:bulk)?(walk|get|getnext)\' .+\'(udp|tcp|tcp6|udp6):(?:\[[0-9a-f:]+\]|[^:]+):[0-9]+\' \'(.+)\'\]$/m';
```

Comment:
Also, this saves like 150 steps:
```suggestion
        $snmp_query_regex = '/SNMP\[\'.*?snmp(?:bulk)?(walk|get|getnext)\' .+ \'(udp6?|tcp6?):[^\']+\' \'([^\']+)\'\]/m';
```